### ISSUE TITLE
No re-download after history remove/reset

### ISSUE THREAD

**Issue by stueja** [fnep/mtv_dl#41]
tl;dr:  my problem was that I had 2 .History files at that time, one in `cwd` and one in `{dir}` and that mtv_dl deleted the files from the database in `cwd` and not in `{dir}`, because I used `mtv_dl history --remove` and not `mtv_dl history --remove --config=./.mtv_dl.yml`. 

---

I have just stumbled upon this wonderful piece of software and I am testing it right now.
While playing with settings and filters, I had do re-download a tv show, which wasn't possible.
```
[jan@two-arch bin]$ ./mtv_dl download --config=./.mtv_dl.yml --sets=./mtv_tatort.txt
2021-10-21T09:07:46  DEBUG    Opening Filmliste database PosixPath('/home/jan/Videos/Tatort/.Filmliste.0.18.1.sqlite').                                                      
                     DEBUG    Opening History database PosixPath('/home/jan/Videos/Tatort/.History.sqlite').                                                                 
                     DEBUG    Database age is 0:17:00.                                                                                                                       
                     DEBUG    Applying filter: channel=ard, topic=tatort, title!=audiodeskription, age-1d, duration+80m (limit: None)                                        
2021-10-21T09:07:47  DEBUG    Skipping 'Tatort: Altes Eisen' (ARD, 'Tatort: Altes Eisen', 2021-10-20 20:00:00, c80523597cd) (already loaded on 2021-10-21 06:52:14)          
                     DEBUG    Skipping 'Absturz' (ARD, 'Tatort', 2021-10-20 20:10:00, 642cb991142) (already loaded on 2021-10-21 06:53:56)                                   
```
I then removed both shows from the history:
```
[jan@two-arch bin]$ ./mtv_dl history --remove=642cb991142
2021-10-21T09:08:58  INFO     Removed 642cb991142 from history.                                                                                                              
[jan@two-arch bin]$ ./mtv_dl history --remove=c80523597cd
2021-10-21T09:09:09  INFO     Removed c80523597cd from history.                                                                                                              
```
but the result was the same:
```
[jan@two-arch bin]$ ./mtv_dl download --config=./.mtv_dl.yml --sets=./mtv_tatort.txt
2021-10-21T09:09:14  DEBUG    Opening Filmliste database PosixPath('/home/jan/Videos/Tatort/.Filmliste.0.18.1.sqlite').                                                      
                     DEBUG    Opening History database PosixPath('/home/jan/Videos/Tatort/.History.sqlite').                                                                 
                     DEBUG    Database age is 0:19:00.                                                                                                                       
                     DEBUG    Applying filter: channel=ard, topic=tatort, title!=audiodeskription, age-1d, duration+80m (limit: None)                                        
2021-10-21T09:09:15  DEBUG    Skipping 'Tatort: Altes Eisen' (ARD, 'Tatort: Altes Eisen', 2021-10-20 20:00:00, c80523597cd) (already loaded on 2021-10-21 06:52:14)          
                     DEBUG    Skipping 'Absturz' (ARD, 'Tatort', 2021-10-20 20:10:00, 642cb991142) (already loaded on 2021-10-21 06:53:56)                                   
```
Then I tried to reset the history:
```
[jan@two-arch bin]$ ./mtv_dl history --reset
```
but still the same:
```
[jan@two-arch bin]$ ./mtv_dl download --config=./.mtv_dl.yml --sets=./mtv_tatort.txt
2021-10-21T09:09:52  DEBUG    Opening Filmliste database PosixPath('/home/jan/Videos/Tatort/.Filmliste.0.18.1.sqlite').                                                      
                     DEBUG    Opening History database PosixPath('/home/jan/Videos/Tatort/.History.sqlite').                                                                 
                     DEBUG    Database age is 0:19:00.                                                                                                                       
                     DEBUG    Applying filter: channel=ard, topic=tatort, title!=audiodeskription, age-1d, duration+80m (limit: None)                                        
                     DEBUG    Skipping 'Tatort: Altes Eisen' (ARD, 'Tatort: Altes Eisen', 2021-10-20 20:00:00, c80523597cd) (already loaded on 2021-10-21 06:52:14)          
                     DEBUG    Skipping 'Absturz' (ARD, 'Tatort', 2021-10-20 20:10:00, 642cb991142) (already loaded on 2021-10-21 06:53:56)                                   
```

The database itself when used manually is working as expected:
```
[jan@two-arch bin]$ sqlite3 .History.sqlite 
SQLite version 3.36.0 2021-06-18 18:36:39
...
sqlite> .headers on
sqlite> select * from downloaded;
hash|channel|description|region|size|title|topic|website|start|duration|downloaded
c80523597cd8b5e45644c85e64f43f99aecd8e74|ARD|Bei ihren Ermittlungen ...|DE|1135|Tatort: Altes Eisen|Tatort: Altes Eisen|https://www...337|2021-10-21 07:17:28
642cb99114281235c64bf72ef3105097f37d825a|ARD|Eigentlich ...|DE|1262|Absturz|Tatort|https://www...MGY|2021-10-20 20:10:00|5312|2021-10-21 07:19:10
sqlite> select hash from downloaded where hash like 'c80523597cd%';
hash
c80523597cd8b5e45644c85e64f43f99aecd8e74
sqlite> delete from downloaded where hash='c80523597cd8b5e45644c85e64f43f99aecd8e74';
sqlite> select * from downloaded;
hash|channel|description|region|size|title|topic|website|start|duration|downloaded
642cb99114281235c64bf72ef3105097f37d825a|ARD|Eigentlich ...|DE|1262|Absturz|Tatort|https://www...MGY|2021-10-20 20:10:00|5312|2021-10-21 07:19:10
```
